apps/myaccount/src/components/account-recovery/options/sms-recovery/sms-recovery.tsx

```diff
@@ -0,0 +1,429 @@
+            }[];
+            schemas: string[];
+        } = {
+            Operations: [
+                {
+                    op: "replace",
+                    value: {}
+                }
+            ],
+            schemas: [ "urn:ietf:params:scim:api:messages:2.0:PatchOp" ]
+        };
+
+        data.Operations[0].value = {
+            phoneNumbers: mobileType || phoneNumberSchema
+                ? [
+                    {
+                        type: mobileType || phoneNumberSchema.name,
+                        value: mobile
+                    }
+                ]
+                : [ mobile ]
+        };
+
+        updateProfileInfo(data)
+            .then(() => {
+                onAlertFired({
+                    description: t(
+                        "myAccount:components.accountRecovery.SMSRecovery.notifications.updateMobile" +
+                        ".success.description"
+                    ),
+                    level: AlertLevels.SUCCESS,
+                    message: t(
+                        "myAccount:components.accountRecovery.SMSRecovery.notifications.updateMobile.success.message"
+                    )
+                });
+
+                dispatch(getProfileInformation());
+                dispatch(setActiveForm(null));
+            })
+            .catch((error: AxiosError) => {
+                if (error.response && error.response.data && error.response.data.detail) {
+                    onAlertFired({
+                        description: t(
+                            "myAccount:components.accountRecovery.SMSRecovery." +
+                            "notifications.updateMobile.error.description",
+                            { description: error.response.data.detail }
+                        ),
+                        level: AlertLevels.ERROR,
+                        message: t(
+                            "myAccount:components.accountRecovery.SMSRecovery." +
+                            "notifications.updateMobile.error.message"
+                        )
+                    });
+
+                    return;
+                }
+
+                onAlertFired({
+                    description: t(
+                        "myAccount:components.accountRecovery.SMSRecovery." +
+                        "notifications.updateMobile.genericError.description"
+                    ),
+                    level: AlertLevels.ERROR,
+                    message: t(
+                        "myAccount:components.accountRecovery.SMSRecovery." +
+                        "notifications.updateMobile.genericError.message"
+                    )
+                });
+            });
+    };
+
+    /**
+     * This function gets the mobile number from the response passed as the argument
+     * and assigns it to mobile and editedMobile.
+     * @param response - response as the parameter.
+     * Temporarily the first mobile type element in the phoneNumbers array is shown.
```

Comment:
Let's show additional info with the `@remarks` tag.

```suggestion
     * @remarks
     * Temporarily the first mobile type element in the phoneNumbers array is shown.
```

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/6418/commits/c189193598ea63ba97e745308cac1b8b89ce7403